elastic/integrations
# Azure OpenAI V1 GA

- Enhancement

This PR has the following changes,

- Remove the signal specific dashboard.
- Includes overview dashboard.
- Remove the RAI requests parameters which are legacy fields that doesn't have any metrics values.
- Replace the content filter requests total with the total tokens in the overview dashboard.
- Make the integrations **GA**.

## Checklist



## How to test this PR locally

- Install the Azure openAI integrations.
- Add the eventhub and azure monitor in the Azure OpenAI service.
- Add the configurations for eventhub and azure monitor in the integration.
- Verify the data is loading for the logs and metrics.
- Check the dashboard values.




## Screenshots

### Integrations List
<img width="1559" alt="GA" src="https://github.com/elastic/integrations/assets/101238137/df6ce7da-b86a-4736-bb97-80e330276109">

### Dashboard
 
<img width="1723" alt="azure_open_ai_overview" src="https://github.com/elastic/integrations/assets/101238137/8377a4ff-5684-4967-8443-26c00627cbd8">
